src/module.c

```diff
@@ -5800,6 +5800,7 @@ void moduleReleaseGIL(void) {
  *                               Notice, when this event fires, the given key
  *                               can not be retained, use RM_CreateStringFromString
  *                               instead.
+ *  - REDISMODULE_NOTIFY_MODULE: Module types events
```

Comment:
since this is just documentation, we can re-order these and put it next to the other types.